itl/plugins-contrib.d/network-components.conf

```diff
@@ -833,3 +833,98 @@ object CheckCommand "printer_health" {
+		}
+		"--port" = {
+			value = "$fiberchannel_brocade_port$"
+			description = "The SNMP port to use (default: 161)"
+		}
+		"--context" = {
+			value = "$fiberchannel_brocade_context$"
+			description = "SNMPv3 context name"
+		}
+		# It seems --fc-port parameter is not working using its short form
+		"-P" = {
+			value = "$fiberchannel_brocade_fc_port$"
+			description = "Port number as shown in the output of `switchshow`. Can't combine with --systeminfo parameter"
+		}
+		"--systeminfo" = {
+			set_if = "$fiberchannel_brocade_systeminfo$"
+			description = "Get global data like boot date, overall status, reachability"
+		}
+		"--sensor" = {
+			set_if = "$fiberchannel_brocade_sensor$"
+			description = "Additional to --systeminfo. Status of powersupply, fans and temp sensors"
+		}
+		"--global" = {
+			set_if = "$fiberchannel_brocade_global$"
+			description = "Global data like boot date, overall status"
+		}
+		"--performancedata" = {
+			value = "$fiberchannel_brocade_perfdata$"
+			description = "Print performance data of the selected FC port specified via -P parameter"
+		}
+		"--secname" = {
+			value = "$fiberchannel_brocade_secname$"
+			description = "The SNMPv3 securityName for the USM security model"
+		}
+		"--authpassword" = {
+			value = "$fiberchannel_brocade_authpassword$"
+			description = "The authentication password for SNMPv3"
+		}
+		"--authproto" = {
+			value = "$fiberchannel_brocade_authprotocol$"
+			description = "The authentication protocol for SNMPv3 (md5|sha)"
+		}
+		"--privpasswd" = {
+			value = "$fiberchannel_brocade_privpassword$"
+			description = "The password for SNMPv3 authPriv security level"
+		}
+		"--privproto" = {
+			value = "$fiberchannel_brocade_privprotocol$"
+			description = "The private protocol for SNMPv3 (des|aes)"
+		}
+		"--sfptemp" = {
+			set_if = "$fiberchannel_brocade_sfptemp$"
+			description = "Checks the temperature of all SFPs"
+		}
+		"--sfptemp_warn" = {
+			value = "$fiberchannel_brocade_sfptemp_warning$"
+			description = "This is the warning offset the critical temperature as delivered by the system. Default is 10 Celsius. Must be used with --sfptemp parameter"
+		}
+		"--maxsfptemp" = {
+			value = "$fiberchannel_brocade_sfptemp_critical$"
+			description = "Maximum temperature for SFPs. If not set it will be taken from your switch"
+		}
+		"--allports" = {
+			set_if = "$fiberchannel_brocade_allports$"
+			description = "Default is only to show ports which are too hot. With this flag all ports will be shown. Must be used with --sfptemp"
+		}
+		"--multiline" = {
+			set_if = "$fiberchannel_brocade_multiline$"
+			description = "Multiline output in overview. This mean technically that a multiline output uses a HTML <br> for the GUI instead of be aware that your messing connections (email, SMS...) must use a filter to file out the <br>. A sed oneliner will do the job."
+		}
+	}
+	vars.fiberchannel_brocade_hostname = "$check_address$"
```

Comment:
Please separate arguments and vars by an empty line.

Reply:
Fixed, my mistake :/